apix-impl/52/src/main/java/eu/xenit/apix/alfresco52/CommentServiceImpl52.java

```diff
@@ -0,0 +1,34 @@
+package eu.xenit.apix.alfresco52;
+
+import com.github.dynamicextensionsalfresco.osgi.OsgiService;
+import eu.xenit.apix.alfresco.ApixToAlfrescoConversion;
+import eu.xenit.apix.alfresco.comments.CommentService;
+import eu.xenit.apix.comments.Comment;
+import eu.xenit.apix.content.IContentService;
+import eu.xenit.apix.node.INodeService;
+import eu.xenit.apix.permissions.IPermissionService;
+import java.util.Map;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service("eu.xenit.apix.comments.CommentService")
+@OsgiService
+public class CommentServiceImpl52 extends CommentService {
+
+    @Autowired
+    public CommentServiceImpl52(org.alfresco.repo.forum.CommentService commentService,
+            IContentService contentService, INodeService nodeService,
+            IPermissionService permissionService,
+            ApixToAlfrescoConversion apixConverter) {
+        super(commentService, contentService, nodeService, permissionService, apixConverter);
+    }
+
+    @Override
+    protected Comment setPermissions(org.alfresco.service.cmr.repository.NodeRef documentNode,
+            org.alfresco.service.cmr.repository.NodeRef commentNodeRef, Comment targetComment) {
```

Comment:
Is there a reason why in `CommentServiceImpl50` and `CommentServiceImpl51` `org.alfresco.service.cmr.repository.NodeRef` is imported but in `CommentServiceImpl52`, `CommentServiceImpl60` and `CommentServiceImpl61` ir isn't?